package.json

```diff
@@ -66,6 +66,7 @@
         "@sentry/browser": "^7.0.0",
         "@sentry/tracing": "^7.0.0",
         "@testing-library/react-hooks": "^8.0.1",
+        "@vector-im/compound-design-tokens": "https://github.com/vector-im/compound-design-tokens#develop",
```

Comment:
Yarn git dependencies using tags/branch refs is troublesome, it overly caches even if the lockfile has changed sha. Can we avoid this please? Its likely to cause headaches.

Reply:
What else do you recommend ?

Reply:
NPM releases, other parties building Element Web also had issues with git dependencies in the past and we had to switch to npm.

E.g. https://github.com/vector-im/element-web/issues/21166 & 

<img width="962" alt="image" src="https://github.com/matrix-org/matrix-react-sdk/assets/2403652/ce664884-3200-40bd-92fb-4e81875ee789">